recipes/sphinxcontrib-plantuml/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "sphinxcontrib-plantuml" %}
+{% set version = "0.25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinxcontrib-plantuml-{{ version }}.tar.gz
+  sha256: 8fde531d92d1cfc2817fe3647b3f2d07e76682c4a84889c04a53e831f7c54432
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```

Should be the as run.

```suggestion
    - python >=3.8
```